includes/commands/class-mu-migration-base.php

```diff
@@ -8,14 +11,14 @@
 	/**
 	 * Holds the command arguments
 	 *
-	 * @var Array
+	 * @var array
```

Comment:
Since we're doing this to adhere WP Standards, let's keep it as it is (long format) since it's how WP uses it.